types/react/index.d.ts

```diff
@@ -2789,7 +2789,7 @@ declare namespace React {
         name?: string;
         required?: boolean;
         size?: number;
-
+        value?: string | string[] | number;
```

Comment:
Not sure about the use case of an array of strings, but that was the previous definition of the value prop:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/fec422d2266ff5041bbc8082ca4c1d675f83efed/types/react/index.d.ts#L2449